frontend/app/views/fragments/form/nameDetailFields.scala.html

```diff
@@ -21,7 +16,7 @@
                    * same size limit is imposed here.
                    *@
                    maxlength="30"
-                   value="@firstName"
+                   value="@privateFields.flatMap(_.firstName)"
```

Comment:
Strictly speaking I don't need `.mkString` here - though I realise I've been inconsistent and used it in `stripeCheckout.scala.html`! Twirl renders `None` options as the empty string, and `Some[String]` objects with just the string content. The code I'm replacing with this commit was already taking advantage of this in fact.

Here's an example of this running OK in dev:

![image](https://cloud.githubusercontent.com/assets/52038/23896825/a6df9ce4-08a2-11e7-8460-cbe4f0eaaafa.png)

You can see there's no `Some(...)` wrapper in the page source:

![image](https://cloud.githubusercontent.com/assets/52038/23896861/c13c4b46-08a2-11e7-86b6-3e16bfff2f90.png)

However, I did use `.mkString` superfluously in `stripeCheckout.scala.html`, so I'll remove that.